test/commands/create/create.test.ts

```diff
@@ -93,7 +93,6 @@ describe('create', () => {
             expect(velocitasConfig.getPackages()[0].version).to.be.equal(TEST_MAIN_PACKAGE_VERSION);
             expect(velocitasConfig.getPackages()[1].repo).to.be.equal(TEST_PACKAGE_URI);
             expect(velocitasConfig.getPackages()[1].version).to.be.equal(TEST_PACKAGE_VERSION);
-            expect(velocitasConfig.getVariableMappings().get('language')).to.be.equal('test');
```

Comment:
if we can get rid of this than you also may adapt the velocitasConfigMock inside mockConfig.ts

Reply:
Done.